src/main/webapp/app/quiz-statistics/quiz-statistic/quiz-statistic.component.ts

```diff
@@ -339,45 +341,22 @@ export class QuizStatisticComponent implements OnInit, OnDestroy, DataSetProvide
     }
 
     /**
-     * got to the Template with the next QuizStatistic -> the first QuizQuestionStatistic
+     * got to the template with the next QuizStatistic -> the first QuizQuestionStatistic
      * if there is no QuizQuestionStatistic -> go to QuizPointStatistic
      */
     nextStatistic() {
+        // go to quiz-statistic if the position = last position
         if (this.quizExercise.quizQuestions === null || this.quizExercise.quizQuestions.length === 0) {
-            this.router.navigate([
-                '/quiz/:quizExerciseId/quiz-point-statistic',
-                {
-                    quizExerciseId: this.quizExercise.id,
-                },
-            ]);
+            this.router.navigateByUrl('/quiz/' + this.quizExercise.id + '/quiz-point-statistic');
         } else {
+            // go to next question-statistic
             const nextQuestion = this.quizExercise.quizQuestions[0];
-            if (nextQuestion.type === QuizQuestionType.MULTIPLE_CHOICE) {
-                this.router.navigate([
-                    'quiz/:quizId/multiple-choice-question-statistic/:questionId',
-                    {
-                        quizId: this.quizExercise.id,
-                        questionId: nextQuestion.id,
-                    },
-                ]);
-            } else if (nextQuestion.type === QuizQuestionType.DRAG_AND_DROP) {
-                this.router.navigate([
-                    'quiz/:quizId/drag-and-drop-question-statistic/:questionId',
-                    {
-                        quizId: this.quizExercise.id,
-                        questionId: nextQuestion.id,
-                    },
-                ]);
-            } else if (nextQuestion.type === QuizQuestionType.SHORT_ANSWER) {
-                this.router.navigate([
-                    'quiz/:quizId/short-answer-question-statistic/:questionId',
-                    {
-                        quizId: this.quizExercise.id,
-                        questionId: nextQuestion.id,
-                    },
-                ]);
-            } else {
-                console.log('Question type not yet supported: ' + nextQuestion);
+            if (nextQuestion.type === this.MULTIPLE_CHOICE) {
+                this.router.navigateByUrl('/quiz/' + this.quizExercise.id + '/multiple-choice-question-statistic/' + nextQuestion.id);
```

Comment:
Can you use string interpolation here. Like so: ``/quiz/${this.quizExercise.id}/multiple-...``